### DISCUSSION TITLE
Suggestions keep suggesting me to follow the same github users for over a year already

### DISCUSSION THREAD

**Discussion by Roms1383** [community/community#57583] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi there and thanks for Github and all its features in general.

There's just one point I quite don't get up to this day : in "Suggestions" feeds Github keeps suggesting me to follow the same users over and over for I don't know how long, but probably at least over a year already. Most of them are either (more or less) related to topics I'm interested to or, weirdly enough, former coworkers.

I have personally no interest in following these users, and I don't get why they still show up in the feeds every single day.
What I would suggest would be either :
- add a button like "I'm not interested" or something like that.
- have the algorithm toss the suggestions after these users have been repeatedly shown for a few weeks and no action has been taken (which probably means the person has no interest in following these users).

Thanks for taking the time to read me.

**Comment by scottajames** [comment]
Have you tried changing your personalization settings to see if it changes the people you get recommended?

**Reply by Roms1383** [reply]
Just browsed settings after seeing your message but couldn't find anything relevant there.
